configs/fine-tune-llama2/gce.yaml

```diff
@@ -0,0 +1,10 @@
+head_node_type:
+  name: head_node_type
+  instance_type: n1-standard-4
+
+worker_node_types:
+- name: gpu_worker
+  instance_type: g2-standard-16-nvidia-l4-1
```

Comment:
The specs make sense. Can you still ACC here that you've tested these configurations? Meaning that you've run a full-parameter FT job of 7B, 13B and 70B for the AWS and GCE yamls here, please?

Reply:
1. This compute config doesn't work for 70B for sure because you need g5.48xlarge for that. The 2 configs are only for 7B. In the README, we do ask people to switch to different instance types manually if they want to train for 13B and 70B.
2. For AWS, I can use g5.4xlarge for 7b, g5.12xlarge for 13b, g5.48xlarge for 70B. I'm not sure about GCE, the old template doesn't say anything about what instance types to use for 13b and 70b. Can you advice?



RE: 1, I've kicked off full parameter again. Let's see how it goes.
RE: 2. Let's not block on GCE yaml because AIOA will only work on AWS for now. Let's fix it after we merge it.

Reply:
Sure thing! Please post a screenshot of the successful training from "RE: 1"

Reply:
6 runs were all successful: (7b, 13b, 70b) x (lora, without lora)
- 7b full parameter
<img width="780" alt="Screenshot 2024-01-15 at 9 28 31 PM" src="https://github.com/anyscale/templates/assets/9677264/043bae06-3361-4b22-9a50-af5e59b5f0ea">

-7b lora
<img width="1352" alt="Screenshot 2024-01-15 at 9 28 45 PM" src="https://github.com/anyscale/templates/assets/9677264/3e3b517c-8b8d-4944-9443-2c21ef592a59">

-13b full parameter

<img width="1347" alt="Screenshot 2024-01-15 at 9 29 14 PM" src="https://github.com/anyscale/templates/assets/9677264/57a67620-acb2-486a-9626-b43599810673">


-13b lora
<img width="1361" alt="Screenshot 2024-01-15 at 9 29 27 PM" src="https://github.com/anyscale/templates/assets/9677264/56c99240-98fc-4614-89b0-1ffd2b28eff0">


-70b full parameter
<img width="1365" alt="Screenshot 2024-01-16 at 9 06 34 AM" src="https://github.com/anyscale/templates/assets/9677264/ba477ae2-b5e0-4883-8d87-3be714626e83">
 
<img width="1394" alt="Screenshot 2024-01-16 at 9 06 38 AM" src="https://github.com/anyscale/templates/assets/9677264/5255e3a2-81fa-4332-9293-fc4b7643f49d">

-70b lora
<img width="1417" alt="Screenshot 2024-01-16 at 9 07 15 AM" src="https://github.com/anyscale/templates/assets/9677264/8051d6d7-e230-49f3-8bb5-b3eec05c87ab">

<img width="1408" alt="Screenshot 2024-01-16 at 9 07 20 AM" src="https://github.com/anyscale/templates/assets/9677264/985b1464-567e-44fa-b19f-ece48af905a1">

Reply:
BTW, I also did a 7b full parameter run using the gcp yaml. It worked as well. The only issue is that the l4 gpu seems much harder to get.
<img width="1383" alt="Screenshot 2024-01-16 at 9 13 33 AM" src="https://github.com/anyscale/templates/assets/9677264/65ef7a10-24fe-4abd-8fe2-2ffe7f575598">

Reply:
Thanks for doing all this work. Great to see that everything works as expected 👍